monai/transforms/utils.py

```diff
@@ -392,3 +393,20 @@ def generate_spatial_bounding_box(img, select_fn=lambda x: x > 0, channel_indexe
         box_start.append(max(0, np.min(nonzero_idx[i]) - margin))
         box_end.append(min(data.shape[i], np.max(nonzero_idx[i]) + margin + 1))
     return box_start, box_end
+
+
+def get_largest_connected_component_mask(img):
+    """
+    Gets the largest connected component mask of an image.
+
+    Args:
+        img (ndarrary): Image to get largest connected component from.
+    """
+    labels = measure.label(img > 0)
```

Comment:
`measure.label` has an important parameter: `connectivity`, I think we should support to config it. It affects the connected area a lot. What do you think?

Reply:
sure, will do